recipe/build.sh

```diff
@@ -16,7 +16,7 @@ cmake \
 cmake --build . --parallel ${CPU_COUNT} --verbose
 
 # test
-if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR}" != "" ]]; then
+if [[ ("${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR}" != "") && "$target_platform" != "osx-64"  ]]; then
```

Comment:
@rhyspoulton, I advised you to use `target_platform`, but I think for this case the best solution might actually to check `"$(uname)" == "Darwin"` or similar, since this is likely all of macOS, not just `osx-64`.

Reply:
Yes, that makes sense. I have updated this.